.travis.yml

```diff
@@ -1,7 +1,8 @@
-sudo: false
+sudo: required
 language: node_js
 node_js:
 - "stable"
+- "0.12"
 - "0.10"
```

Comment:
I don't know if there's any public stats, but I doubt many are using 0.10 on GAE. @JustinBeckwith what do you think?